cmd/experimental/kjobctl/apis/v1alpha1/volumebundle_types.go

```diff
@@ -17,11 +17,31 @@ limitations under the License.
 package v1alpha1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // VolumeBundleSpec defines the desired state of VolumeBundle
-type VolumeBundleSpec struct{}
+// +kubebuilder:validation:XValidation:rule="self.mountPoints.map(x, x.name in self.volumes.map(y, y.name))", message="mountPoint name must match a volume name"
+type VolumeBundleSpec struct {
+	// Volumes is a set of volumes that will be added to all pods of the job.
```

Comment:
Almost forgot. We use the json names in API comments. We probably slipped some violations at some point. Feel free to open separate PRs if you find any.

```suggestion
	// volumes is a set of volumes that will be added to all pods of the job.
```